lib/rules/block-indentation.js

```diff
@@ -101,6 +135,9 @@ module.exports = class BlockIndentation extends Rule {
         '  * boolean - `true` to enable 2 space indentation',
         '  * numeric - the number of spaces to require',
         '  * "tab" - usage of one character indentation (tab char)',
+        '  * object - An object with the following keys:',
+        '    * `indentation: <numeric>` - number of spaces to indent',
+        '    * `ignoreComments: <boolean>` - skip comment indentation',
```

Comment:
This should match the comment in `block-indentation.js` (should also mention the default values here).

And the rule documentation file needs to be updated to mention this new option too.